app/controllers/AdminController.php

```diff
@@ -421,33 +434,36 @@ public function getExperimentsOfTimeRange()
     }
 
 	public function createSSH(){
-		$newToken = AdminUtilities::create_ssh_token_for_gateway(null);
+        $description = Input::get("description");
+        if($description == "Default SSH Key"){
```

Comment:
I don't think this check is necessary.

On the other hand, it might be nice to prevent admins from creating two keys with the same description.

Reply:
Admins cannot create two keys with the same description. I'm doing this on client side using jQuery.